test/components/views/settings/tabs/user/AppearanceUserSettingsTab-test.tsx

```diff
@@ -31,7 +32,10 @@ describe("AppearanceUserSettingsTab", () => {
     });
 
     it("should render", () => {
-        const { asFragment } = render(<AppearanceUserSettingsTab />);
+        const { asFragment } = render(
+            <AppearanceUserSettingsTab />,
+            renderWithClientContextOptions(MatrixClientPeg.safeGet()),
```

Comment:
feels like we should stash the client returned by `stubClient` in the test class rather than introduce new references to MatrixClientPeg. YMMV though.